demos/common/python/pipelines/async_pipeline.py

```diff
@@ -17,6 +17,67 @@
 import logging
 import threading
 from collections import deque
+from typing import Dict, Set
+
+
+def parse_devices(device_string):
+    colon_position = device_string.find(':')
+    if colon_position != -1:
+        device_type = device_string[:colon_position]
+        if device_type == 'HETERO' or device_type == 'MULTI':
+            comma_separated_devices = device_string[colon_position + 1:]
+            devices = comma_separated_devices.split(',')
+            for device in devices:
+                parenthesis_position = device.find(':')
+                if parenthesis_position != -1:
+                    device = device[:parenthesis_position]
+            return devices
+    return (device_string,)
+
+
+def parse_value_per_device(devices: Set[str], values_string: str)-> Dict[str, int]:
+    """Format: <device1>:<value1>,<device2>:<value2> or just <value>"""
+    values_string_upper = values_string.upper()
+    result = {}
+    device_value_strings = values_string_upper.split(',')
+    for device_value_string in device_value_strings:
+        device_value_list = device_value_string.split(':')
+        if len(device_value_list) == 2:
+            if device_value_list[0] in devices:
+                result[device_value_list[0]] = int(device_value_list[1])
+        elif len(device_value_list) == 1 and device_value_list[0] != '':
+            for device in devices:
+                result[device] = int(device_value_list[0])
+        elif device_value_list[0] != '':
+            raise RuntimeError(f'Unknown string format: {values_string}')
+    return result
+
+
+def get_user_configs(flags_d: str, flags_nstreams: str, flags_nthreads: int)-> Dict[str, str]:
```

Comment:
I'd prefer the old name, `get_plugin_configs`.

Reply:
The term plugin is hidden in IE implementation now. I don't thing it is a good idea to mention it.

Reply:
We could (and should) use brand new implementation in all py demos now. And not all demo have a pipeline approach and configure the IE manually.

Reply:
Straightforward pipelines wont' utilize more than one stream or req. `get_user_configs()` is supposed to be used with `AsyncPipeline` similar to C++.